vendors/circom/prover_main.cc

```diff
@@ -166,7 +166,10 @@ void CreateProof(const base::FilePath& zkey_path,
     zk::r1cs::groth16::PreparedVerifyingKey<Curve> prepared_verifying_key =
         std::move(proving_key).TakeVerifyingKey().ToPreparedVerifyingKey();
     CHECK(zk::r1cs::groth16::VerifyProof(prepared_verifying_key, proof,
-                                         public_inputs));
+                                         public_inputs))
+        << "If you encounter this error with `--config cuda`, it means your "
+           "GPU RAM is insufficient for running Icicle. Please disable "
+           "`--config cuda` and try running again.";
```

Comment:
I don't think disabling cuda build is the soultion. I think we should guide them to use GPU with more memory.